README.md

```diff
@@ -185,49 +254,75 @@ This step will do a set of tasks:
 
 _You would only be interested here at minima in the result downloading_
  
-...and post processing tasks.
+...and postprocessing tasks. The main postprocessing steps for each pipeline are as followed:
+
+#### CN
 
-> Unfortunately for now the postProcessing tasks are not all made to be easily run outside of the CCLE pipeline. Most of them are in R and are run with the Rpy2 tool.
+`copynumbers.py` contains the main postprocessing function (`postProcess()` and their wrappers for internal use) responsible for postprocessing segments and gene-level CN files.
 
-So amongst these functions, some of them might be of a lesser interest to an external user. The most important ones for each pipelines are:
+#### Mutation
+
+`postProcess()` in `mutations.py` is responsible for postprocessing aggregated MAF files. Here, rather than rerunning the entire analysis, because we know we are adding only WES samples, we can download the previous release's MAF, add the samples, update any annotations, and perform any global filters at the end.
```

Comment:
this is not what we do actually we rerun everything everytime for the postprocessing. It is actually bad practice for a pipeline to directly use results from run n-1.